lib/launcher.js

```diff
@@ -13,12 +13,29 @@ class SeleniumStandaloneLauncher {
     }
 
     onPrepare (config) {
+        const launcher = this
+
         this.seleniumArgs = config.seleniumArgs || {}
         this.seleniumInstallArgs = config.seleniumInstallArgs || {}
         this.seleniumLogs = config.seleniumLogs
         this.logToStdout = config.logToStdout
+        this.skipSeleniumInstall = !!config.skipSeleniumInstall
+
+        if (this.skipSeleniumInstall) {
+            return this._startSelenium(this.seleniumArgs)
+        }
+
+        return this._installSeleniumDependencies(this.seleniumInstallArgs).then(() => launcher._startSelenium(launcher.seleniumArgs))
```

Comment:
let's use [`util.promisify`](https://nodejs.org/dist/latest-v10.x/docs/api/util.html#util_util_promisify_original) instead of `_startSelenium`